Automattic/sensei
# Improve question block navigation

Depends on  #3967

### Changes proposed in this Pull Request

* Automatically insert an empty question block at the end of the quiz
  — inserts a new empty block if the quiz block is selected and the last question is not empty (has a title)
  — removes auto-inserted empty block if the quiz block is not focused
* Jump to question description on Enter keypress in question title
* Delete question block on backspace in empty question title 
  — :question: Should the block not be deleted if it has content (description, answers)?
* Remove rich-text formatting from question title
  — I don't think we wanted to add this for now
* Ensure first question block is focused when adding a quiz

### Testing instructions

* Add a quiz block. Add questions
* Check the quiz block always has an empty question at the end when selected
--
* Type in a question title, press `Enter`
* Check the question description block is now selected
--
* Delete a question title and press one more `Backspace`
* Check that the question is removed
--
* Check that in the Course Outline block, auto-inserted lessons in module blocks continue to work as expected


### Screenshot / Video

![QuickTime movie](https://user-images.githubusercontent.com/176949/107822306-2d678d00-6d7e-11eb-8ea3-5d928493f527.gif)
